src/component/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,45 @@
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  list: Todo[];
+};
+
+export const ToDoList: React.FC<Props> = ({ list }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {list.map(todo => (
+        <div
+          data-cy="Todo"
+          className={`todo ${todo.completed && 'completed'}`}
+          key={todo.id}
+        >
+          {/* eslint-disable jsx-a11y/label-has-associated-control  */}
+          <label className="todo__status-label" htmlFor={'' + todo.id}>
+            <input
+              id={'' + todo.id}
+              data-cy="TodoStatus"
+              type="checkbox"
+              className="todo__status"
+              checked={todo.completed ? true : false}
```

Comment:
```suggestion
              checked={todo.completed}
```
why you need ternary here?